.travis.yml

```diff
@@ -17,8 +17,8 @@ install:
 before_script:
 - emacs --version
 script:
-- emacs --script .emacs.d/init.el
 - emacs --script .emacs.d/test/lisp/run-tests.el
+- emacs --script .emacs.d/init.el
```

Comment:
Oh, my apologies, I thought there was a `~/` at the start of this path and I was very confused.